### ISSUE TITLE
Replicate RECAP PDF uploads to subdockets

### ISSUE THREAD

**Issue by albertisfu** [freelawproject/courtlistener#4826]
In https://github.com/freelawproject/courtlistener/pull/4665 attachment pages that belong to doppeldockets are replicated across all subdockets.

This approach should also be applied to PDF uploads.

**Comment by mlissner** [comment]
There's an opposite direction to this, isn't there? For example, if we have docket ABC with two PDFs associated with it, then we get a doppeldocket in the same case, shouldn't we copy over the two PDFs to the doppeldocket?

**Comment by albertisfu** [comment]
Do you mean that initially we have, for example, a case with many entries and documents already merged into it, and at that moment, we don’t know it’s a doppel docket? Then, we get another docket that’s a subdocket of the initial cases so we need to merge the entries and PDFs from the initial docket into the new one?

If that’s correct, I have some questions:

Should we merge all the content from the initial case into all the other subdockets, or only certain entries/PDFs?
For instance, I recall recently reviewing this RECAP email while debugging a different issue:

-------------------------------------
**Notice of Electronic Filing**

The following transaction was entered on 12/9/2024 at 10:33 AM EST and filed on 12/9/2024
Case Name:	In Re: Terrorist Attacks on September 11, 2001
Case Number:	[1:03-md-01570-GBD-SN](https://ecf.nysd.uscourts.gov/cgi-bin/DktRpt.pl?242129)
Filer:	
Document Number:	No document attached
Docket Text:
***NOTICE TO COURT REGARDING PROPOSED DEFAULT JUDGMENT. Document No. (895 in 1:15-cv-09903-GBD-SN, 10586 in 1:03-md-01570-GBD-SN) Proposed Default Judgment, was reviewed and approved as to form. Filed In Associated Cases: 1:03-md-01570-GBD-SN, 1:15-cv-09903-GBD-SN(nd)

Case Name:	Burnett et al v. The Islamic Republic of Iran et al
Case Number:	[1:15-cv-09903-GBD-SN](https://ecf.nysd.uscourts.gov/cgi-bin/DktRpt.pl?451406)
Filer:	
Document Number:	No document attached
Docket Text:
***NOTICE TO COURT REGARDING PROPOSED DEFAULT JUDGMENT. Document No. (895 in 1:15-cv-09903-GBD-SN, 10586 in 1:03-md-01570-GBD-SN) Proposed Default Judgment, was reviewed and approved as to form. Filed In Associated Cases: 1:03-md-01570-GBD-SN, 1:15-cv-09903-GBD-SN(nd)

-------------------------------------


In this case, the entry appears to be shared between the two cases. I assume that shared entries make these two cases doppel dockets and related to each other. Here, it's a minute entry, but there should be other entries that contain PDFs.

However, I reviewed the first documents of each case, and they are different. This makes me wonder if this is a different type of doppeldocket where only a subset of entries is shared? If that’s correct, I don’t think it’s viable to copy all the content from one docket to another. Perhaps an additional step would be required to check which `pacer_doc_ids` are common between the two cases and copy only those PDFs. However, this approach will only work if the subdocket already has the entries and attachment data merged.

**Comment by mlissner** [comment]
Yeah, that example is a bit strange, but even with that, I think the approach would be the same:

 - Do *not* copy over entries
 - *Do* copy over PDFs.

As you've observed, some of the entries will be shared, but not others. But the goal is to make each docket as complete as possible, which will mean copying PDFs bi-directionally.
